### PR TITLE
examples

### PR DESCRIPTION
**Pull request by vxav**
Add examples on how to create clusters.

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] Update Lastpass values if required.

### CODE REVIEW

File: examples/cluster-default/02_apps.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      name: cluster01-user-values
+      namespace: org-multi-project
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/managed-by: flux
+  name: cluster01-default-apps
```
Comment (erkanerol) [inline examples/cluster-default/02_apps.yaml:27]: After moving CPI/CSI/CNI to this app, I don't think we should have default-apps-cloud-director here.

Comment (vxav) [reply examples/cluster-default/02_apps.yaml:27]: I see what you mean, however, this is already the version with CSI/CPI/CNI embedded in the cluster chart.
The idea is to give GS users/customers a template to create a cluster from A to Z easily.
I added a readme at the root of the examples folder to give a bit of context. wdyt?
[9b19e4f](https://github.com/giantswarm/cluster-cloud-director/pull/106/commits/9b19e4fa20bc993ee1265b431355307db42b68db)

**Review by glitchcrab — APPROVED** [review]
seems reasonable to me :ok_hand:

File: examples/cluster-default/02_apps.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/managed-by: flux
+  name: cluster01
+  namespace: org-multi-project
+spec:
+  name: cluster-cloud-director
+  namespace: org-multi-project
+  version: 0.7.1
```
Comment (kopiczko) [inline examples/cluster-default/02_apps.yaml:12]:
```suggestion
  version: 0.8.0
```

File: examples/cluster-advanced/02_apps.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+    giantswarm.io/managed-by: flux
+  name: cluster02
+  namespace: org-multi-project
+spec:
+  name: cluster-cloud-director
+  namespace: org-multi-project
+  version: 0.7.1
```
Comment (kopiczko) [inline examples/cluster-advanced/02_apps.yaml:12]:
```suggestion
  version: 0.8.0
```

File: examples/cluster-advanced/01_configmaps.yaml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+    osUsers:
+    - name: "root"
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
+      sshAuthorizedKeys:
+      - "ssh public key"
+    - name: "johndoe"
+      sudo: "ALL=(ALL) NOPASSWD:ALL"
+      sshAuthorizedKeys:
+      - "ssh public key"
```
Comment (kopiczko) [inline examples/cluster-advanced/01_configmaps.yaml:72]: I'd get rid of that. I'm not even sure if this works and we should remove this setting I think.

Comment (vxav) [reply examples/cluster-advanced/01_configmaps.yaml:72]: it does work, I think it shows how to add users for debugging

**Pull request merged**